### PR TITLE
Hold old

### PR DESCRIPTION
**Pull request by HansOlsson**
Put old MCPs on hold and add some old (still relevant) public material.

### CODE REVIEW

**Review by henrikt-ma — COMMENTED** [review]
I wonder if old and not yet accepted MCPs (thinking of MCP-0012) should really be present on _master_?

**Comment by HansOlsson** [comment]
> I wonder if old and not yet accepted MCPs (thinking of MCP-0012) should really be present on _master_?

I wasn't sure either.
At first I considered having yet another branch, but I decided against it since we already have too many branches making the repository look confusing - especially if we want to add additional old MCPs.

Thus I preliminary concluded that it's fine to have not yet accepted MCP documents in the main repository, _provided that they are not being worked on_.

**Comment by henrikt-ma** [comment]
> Thus I preliminary concluded that it's fine to have not yet accepted MCP documents in the main repository, _provided that they are not being worked on_.

Sounds like a good topic for a regular phone meeting.

File: RationaleMCP/ReadMe.md [hunk 1]
```diff
@@ -42,16 +42,18 @@ but the rest of the development on a branch/pull-request before being accepted.
...
-- MCP0012 Calling blocks as functions
-- MCP0011 Allow user-defined functions in reductions
-- MCP0010 Adding guards to reductions
-- MCP0009 Undefined modification
-- MCP0008 Custom annotations
-- MCP0007 Match expressions
-- MCP0006 Atomic blocks
-- MCP0005 Equivalent parameters
-- MCP0004 Handling uncertainties (need to consider MCP0008)
+- ~~MCP0013~~ Introducing polymorphic functions
```
Comment (beutlich) [inline RationaleMCP/ReadMe.md:45]: Suspendend MCPs are visually not distinguishable from closed MCPs. I'd prefer to make that more transparent, either by having a proper table with MCP states (Official state are: In Preparation, In Development, Under Evaluation, Accepted, Scheduled, New states are: Suspended, Implemented) or by having separate headings for each state.

Comment (HansOlsson) [reply RationaleMCP/ReadMe.md]: I partially implemented this.
"Added in " for the ones that are implemented, and "On Hold" for the ones on hold, but just "Active" for the rest.
Can later make that more detailed.

**Comment by HansOlsson** [comment]
> > Thus I preliminary concluded that it's fine to have not yet accepted MCP documents in the main repository, _provided that they are not being worked on_.
> 
> Sounds like a good topic for a regular phone meeting.

I have now removed them here.

File: RationaleMCP/ReadMe.md [hunk 2]
```diff
@@ -17,41 +17,46 @@ New MCP should be added to the following list - on the main branch to keep track
...
-- MCP0006 Atomic blocks
-- MCP0005 Equivalent parameters
-- MCP0004 Handling uncertainties (need to consider MCP0008)
-- ~~MCP0003~~ User defined annotations (superseded by MCP0008)
-- MCP0002 Improved class generation
-- MCP0001 Improved parametrization
+
+|Status|Number|Name|Link|
+|------|------|----|----|
+|Active|0038|Initialization of Clocked Partitions|([MCP/0038](https://github.com/modelica/ModelicaSpecification/tree/MCP/0038/RationaleMCP/0038))|
```
Comment (beutlich) [inline RationaleMCP/ReadMe.md:23]: There probably is no more need to have those parentheses surround the MCP/issue link, but that is just cosmetics.

**Review by beutlich — APPROVED** [review]

**Pull request merged**